.changeset/nice-fireants-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
+---
```

Comment:
Shouldn't we test the disabled? If a trigger is disabled we want the tooltip not to show.

Reply:
oh no i don't think that's right francis. It's actually something that needs to be tested. We absolutely need to show a tooltip on a disabled button. It's one of the most frequently used pattern by design

Reply:
> oh no i don't think that's right francis. It's actually something that needs to be tested. We absolutely need to show a tooltip on a disabled button. It's one of the most frequently used pattern by design

I think this got lost on translation, a tooltipTrigger can be disabled. But you can have a disabled trigger that shows a tooltip since the "trigger" is nested inside the trigger.

Reply:
https://wl-orbiter-website.netlify.app/?path=/docs/tooltip--default-story#disabled